### DISCUSSION TITLE
How do I prevent this and set the height of the image from the url?

### DISCUSSION THREAD

**Discussion by Extravi** [anuraghazra/github-readme-stats#3158] [Q&A] [answered]
![Screenshot_20230827_234650](https://github.com/anuraghazra/github-readme-stats/assets/98912029/c9f8bce5-440e-47bd-975b-af7ded834a9b)

The font and size is off for all of them when i try to make them look the same

```html
            <a href="https://github.com/Extravi/tailsx" target="_blank">
                <img alt="svg" class="github" align="middle" src="https://github-readme-stats.vercel.app/api/pin/?username=Extravi&amp;repo=tailsx&amp;theme=dark&amp;border_color=3c4043" height="150">
            </a>            
```

**Comment by qwerty541** [comment, marked as answer]
Hey, @Extravi! Unfortunately, at the moment it is impossible to achieve a result in which the height and font size of cards with different description lengths are the same. We're missing query parameter for manual setting card height. Feel free to open feature request issue.

<div><img align="center" src="https://github-readme-stats.vercel.app/api/pin/?username=Extravi&amp;repo=tailsx&amp;theme=dark&amp;border_color=3c4043" />
<img align="center" src="https://github-readme-stats.vercel.app/api/pin/?username=Extravi&amp;repo=installer&amp;theme=dark&amp;border_color=3c4043" /></div>

**Comment by Extravi** [comment]
https://github.com/anuraghazra/github-readme-stats/issues/3159
